src/main/java/com/answerdigital/answerking/exception/util/RestResponseEntityExceptionHandler.java

```diff
@@ -25,11 +25,12 @@
 @RestControllerAdvice
 public class RestResponseEntityExceptionHandler {
 
+    @SneakyThrows
```

Comment:
I don't think SneakyThrows is necessary here, or anywhere else it is used within this class! Just want to emphasis I'm not 100% sure though! If there was a specific reason I'd love to learn about it!

```suggestion

```